test/integration/stop/test_stop.py

```diff
@@ -0,0 +1,90 @@
+import os
+import shutil
+import subprocess
+
+from utils import pid_file, run_path, wait_file
+
+
+def app_cmd(tt_cmd, tmpdir_with_cfg, cmd, input):
```

Comment:
Let's avoid code duplication. I this function can be replaced with `run_command_and_get_output`. Please, check.

Reply:
Made it